core/sqf/src/seatrans/hbase-trx/src/main/java/org/apache/hadoop/hbase/client/transactional/TransactionManager.java

```diff
@@ -2149,6 +2236,11 @@ public Integer call() throws CommitUnsuccessfulException, IOException {
                 loopExit = true; 
               } 
               catch (InterruptedException ie) {}
+              catch (IOException e) {
+                 loopExit = true;
+                 LOG.error("Exception at the time of committing DDL transaction", e); 
```

Comment:
If making any additional changes, could we edit this comment to indicate Exception in DML before committing DDL.